### PR TITLE
Add grism19 for NOT

### PR DESCRIPTION
**Pull request by profxj**
As titled.

There is now a grism19 test set in the Dev suite.

### CODE REVIEW

**Comment by profxj** [comment]
Looks like a `hotfix` on `master` creeped in.  Should be fine, but just a head's up.

**Comment by profxj** [comment]
================ 229 passed, 729 warnings in 544.72s (0:09:04) ================

File: doc/wave_calib.rst [hunk 1]
```diff
@@ -109,11 +109,40 @@ calibration frame.
...
+usage
+-----
+
+Here is the usage::
+
+    usage: pypeit_identify [-h] [--lamps LAMPS] [-s] [--wmin WMIN] [--wmax WMAX]
+                           [--slit SLIT] [--det DET] [--rmstol RMSTOL]
+                           arc_file slits_file
+
+    Launch PypeIt identify tool, display extractedMasterArc, and load linelist.Run
```
Comment (rcooke-ast) [inline doc/wave_calib.rst:121]: missing a space: `extracted MasterArc`
Start `Run` on a new line...

**Comment by profxj** [comment]
Running pypeit on not_alfosc/grism19 (ignores masters)  --- PASSED
[INFO]    :: run_pypeit.py 97 main() - Data reduction complete
[INFO]    :: run_pypeit.py 99 main() - Generating QA HTML


Directories:
         Raw data: /data/Projects/Python/PypeIt-development-suite/RAW_DATA/not_alfosc/grism4
    PypeIt output: /data/Projects/Python/PypeIt-development-suite/REDUX_OUT/not_alfosc/grism4

Directory: /data/Projects/Python/PypeIt-development-suite/REDUX_OUT/not_alfosc/grism4
Command line: run_pypeit not_alfosc_grism4.pypeit -o
Wrote: /data/Projects/Python/PypeIt-development-suite/REDUX_OUT/not_alfosc/grism4/QA/MF_A.html
Running pypeit on not_alfosc/grism4 (ignores masters)  --- PASSED
[INFO]    :: run_pypeit.py 97 main() - Data reduction complete
[INFO]    :: run_pypeit.py 99 main() - Generating QA HTML



--- PYPEIT DEVELOPMENT SUITE PASSED 2/2 TESTS (Masters ignored) ---

**Comment by deborape** [comment]
Looks good to me!

**Comment by profxj** [comment]
merging

**Pull request merged**